<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
   <widget class="QFrame" name="red_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>350</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.522388, y1:0.256, x2:0.537, y2:1, stop:0 rgba(200, 0, 0, 255), stop:1 rgba(76, 1, 1, 255));
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="title1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
color: rgb(231, 231, 231);
</string>
     </property>
     <property name="text">
      <string>Hello, Friend!</string>
     </property>
    </widget>
    <widget class="QLabel" name="sub_title1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(231, 231, 231);
background-color: none;</string>
     </property>
     <property name="text">
      <string>Enter your personal details</string>
     </property>
    </widget>
    <widget class="QLabel" name="sub_title2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(231, 231, 231);
background-color: none;</string>
     </property>
     <property name="text">
      <string>and start journey with us</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup_btn_red">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background-color: none;
border: 2px solid rgb(231, 231, 231);
border-radius: 20px;
color: rgb(231, 231, 231);
}
*:hover{
background-color: rgb(151, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>SIGN UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_acc_btn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>460</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{background-color: none;
color: white;
border: none;
border-radius:0;}
*:hover{border-bottom: 1px solid white;}</string>
     </property>
     <property name="text">
      <string>Delete Account</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="signin_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(80, 80, 80)</string>
    </property>
    <property name="text">
     <string>SIGN IN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fb_btn">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border: 1px solid gray;
border-radius: 20px;
}
*:hover{
background-color: #dcdcdc;
}</string>
    </property>
    <property name="text">
     <string>f</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insta_btn">
    <property name="geometry">
     <rect>
      <x>602</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border: 1px solid gray;
border-radius: 20px;
}
*:hover{
background-color: #dcdcdc;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/insta_icon.png</normaloff>../icons/insta_icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="twitter_btn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border: 1px solid gray;
border-radius: 20px;
}
*:hover{
background-color: #dcdcdc;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/twitter_icon.png</normaloff>../icons/twitter_icon.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="sub_title3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(135, 135, 135);</string>
    </property>
    <property name="text">
     <string>or use your user account</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>290</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);
border-radius: 5px;
padding: 10px 12px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>290</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);
border-radius: 5px;
padding: 10px 12px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="user_img">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);
border-radius: 5px;
color: rgb(109, 109, 109)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/user_icon.png</normaloff>../icons/user_icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pass_img">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);
border-radius: 5px;
color: rgb(109, 109, 109)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/key_icon.png</normaloff>../icons/key_icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="signin_btn_red">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>420</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Light</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: #dc143c;
border: 2px solid rgb(231, 231, 231);
border-radius: 20px;
color: rgb(231, 231, 231);
}
*:hover{
background-color: rgb(151, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>SIGN IN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f_pass_btn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>330</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">*{border: none;}
*:hover{color: #dc143c;}</string>
    </property>
    <property name="text">
     <string>Forgot your password?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eye_img">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>290</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);
border-radius: 5px;
color: rgb(109, 109, 109)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/eye-on_icon.png</normaloff>../icons/eye-on_icon.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
